sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -14,7 +14,9 @@ import { TokenCredential } from '@azure/core-auth';
 export type AnalyzeSentimentErrorResult = TextAnalyticsErrorResult;
 
 // @public
-export type AnalyzeSentimentOptions = TextAnalyticsOperationOptions;
+export interface AnalyzeSentimentOptions extends TextAnalyticsOperationOptions {
+    mineOpinions?: boolean;
```

Comment:
we're actually going to go with `includeOpinionMining` first, you can see pythons [ApiView](https://apiview.dev/Assemblies/Review/f74ef8c4777b408f85dab76b8034cb41/5c5ae05321de468aa1c749de2b44e74a?diffRevisionId=86e49dc083f14c55a13f47bb8e769937) for the changes (side note: there's quite a few diffs since the actual api view modeling tool changed it's output, so please ignore those

Reply:
done.